backend/typescript/rest/givecloud.ts

```diff
@@ -0,0 +1,56 @@
+import { Router } from "express";
+import AuthService from "../services/implementations/authService";
+import UserService from "../services/implementations/userService";
+import { sendErrorResponse } from "../utilities/errorResponse";
+import { AuthDTO, Role, UserDTO } from "../types";
+
+const givecloudRouter: Router = Router();
+
+const authDtoToToUserDto = (authDTO: AuthDTO): UserDTO => {
```

Comment:
might be overkill, not sure what we want to return, but ik it's not good to return authDTO given it has some tokens.

Reply:
Yup, makes sense. Technically we only have to return a `201`, but it might be useful for debug purposes to return the `UserDTO` like so